### ISSUE TITLE
[boschshc] wall-thermostat items is not updated automatically 

### ISSUE THREAD

**Issue by ulrgaer** [openhab/openhab-addons#15871]
the wall-thermostat: Items are not updated automatically. the correct value is only updated once after installation of the item or after system restart.

## Expected Behavior
the items value (temperature, humidity) should be update each time when the chancel value is changed.

## Current Behavior
update only once after configuration of the new item or after system restart.

## Possible Solution

no solution known
## Steps to Reproduce (for Bugs)

1. configure items once
2. wait for update, nothing is changing

## Context
only old values (temperature, humidity ) are shown on openHAB 

## Your Environment

* Version used: openHAB 4.0.4
* Environment name and version : Linux/6.1.21-v8+ (arm)
* Operating System and version : 
* systemInfo:
  configFolder: /etc/openhab
  userdataFolder: /var/lib/openhab
  logFolder: /var/log/openhab
  javaVersion: 17.0.7
  javaVendor: Raspbian
  osName: Linux
  osVersion: 6.1.21-v8+
  osArchitecture: arm
  availableProcessors: 4
  freeMemory: 119675496
  totalMemory: 259522560
  startLevel: 100

**Comment by david-pace** [comment]
Thanks for the bug report @ulrgaer. Do you have any other Bosch SHC things/channels that are updated correctly when this happens? If all channels are affected, the root cause might be #15912.

**Comment by ulrgaer** [comment]
Dear David,the description looks like my case. But what can be done?Best UliAm 20.11.2023 um 22:10 schrieb David Pace ***@***.***>:﻿
Thanks for the bug report @ulrgaer. Do you have any other Bosch SHC things/channels that are updated correctly when this happens? If all channels are affected, the root cause might be #15912.

**Comment by david-pace** [comment]
Hi @ulrgaer, I would propose to proceed as follows:

1. Check if you can find the stack trace mentioned in #15912 in your `openhab.log`
2. If you can't find it, wait until this happens again and then check your `openhab.log` again
3. If the stack trace is present, we assume that #15912 is the root cause. Then I would propose to close this issue (since then it would be a duplicate) and that we continue tracking down the problem in #15912.

By the way I am one step further in analyzing the issue in #15912.

**Comment by ulrgaer** [comment]
I'm still waiting for the next occurrence. Feedback traces will come.

**Comment by david-pace** [comment]
There is another issue that potentially causes this problem: https://github.com/openhab/openhab-addons/issues/16174

Both issues are fixed in the mean time and will be available in the next milestone release. We can leave this issue open until we know for sure that it was one of these two issues.

**Comment by david-pace** [comment]
OpenHAB [4.2.0.M1](https://github.com/openhab/openhab-distro/releases/tag/4.2.0.M1) was released and contains both fixes.

@ulrgaer could you please install this milestone and report back whether the problem is solved?

**Comment by ulrgaer** [comment]
david, thanks for the information, can you please help me hoe to install a milestone release without disturbing the actually running installation, thanks, uli

**Comment by david-pace** [comment]
The only clean way I can think of would be to install openHAB on a separate, spare computer or Rasberry Pi. Do you have one lying around by any chance?

If you have one and if you are willing to invest time in a parallel installation, select the following during the openHabian installation in order to get the latest milestone builds: https://community.openhab.org/t/solved-install-milestone-build-on-openhabian/50802/3

If not you can still wait until the next stable release, but this may take a few more weeks or months.

**Comment by ulrgaer** [comment]
Thanks David, will it work on bookworm as well? If yes, may be I can test by next weekend on bookworm, let me see. Uli

**Comment by david-pace** [comment]
If you mean Debian Bookworm, this should work. Just follow the instructions on [this page](https://www.openhab.org/docs/installation/linux.html), specifically

- Package Repository Installation
- Apt-based System
- Testing Release

**Comment by david-pace** [comment]
Since there were no further reports by the OP, I assume that this issue was/is caused by one of the following bugs:
* https://github.com/openhab/openhab-addons/issues/15912
* https://github.com/openhab/openhab-addons/issues/17176

The first bug was fixed a long time ago, and a fix for the second bug is in development. Please track the progress there if applicable. Once a fix is available for https://github.com/openhab/openhab-addons/issues/17176 and it still does not work, feel free to re-open this bug or open a new one.
